packages/oui-inline-adder/README.md

```diff
@@ -0,0 +1,168 @@
+# Inline adder
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+### Basic
+
+```html:preview
+<oui-inline-adder>
+    <oui-inline-adder-row>
+        <oui-inline-adder-field>
+            <oui-field label="Field 1">
+                <input type="text" class="oui-input" name="field1" ng-model="field1" required>
+            </oui-field>
+        </oui-inline-adder-field>
+        <oui-inline-adder-field ng-hide="$ctrl.togglerLoading">
+            <oui-field label="Field 2">
+                <input type="text" class="oui-input" name="field2" ng-model="field2" required>
+            </oui-field>
+        </oui-inline-adder-field>
+    </oui-inline-adder-row>
+</oui-inline-adder>
+```
+
+### Multiple rows
+
+```html:preview
+<oui-inline-adder>
+    <oui-inline-adder-row>
+        <oui-inline-adder-field>
+            <oui-field label="Field 1">
+                <input type="text" class="oui-input" name="field1" ng-model="field1" required>
+            </oui-field>
+        </oui-inline-adder-field>
+        <oui-inline-adder-field>
+            <oui-field label="Field 2">
+                <input type="text" class="oui-input" name="field2" ng-model="field2">
+            </oui-field>
+        </oui-inline-adder-field>
+    </oui-inline-adder-row>
+    <oui-inline-adder-row>
+        <oui-inline-adder-field>
+            <oui-field label="Field 3">
+                <input type="text" class="oui-input" name="field3" ng-model="field3" required>
+            </oui-field>
+        </oui-inline-adder-field>
+        <oui-inline-adder-field>
+            <oui-field label="Field 4">
+                <input type="text" class="oui-input" name="field4" ng-model="field4">
+            </oui-field>
+        </oui-inline-adder-field>
+    </oui-inline-adder-row>
+</oui-inline-adder>
+```
+
+### Adaptive fields
+
+**Note**: Fields with `adaptive` attribute will adapt their size to their content.
+
+```html:preview
+<oui-inline-adder>
+    <oui-inline-adder-row>
+        <oui-inline-adder-field>
+            <oui-field label="Field 1">
+                <input type="text" class="oui-input" name="field1" ng-model="field1" required>
+            </oui-field>
+        </oui-inline-adder-field>
+        <oui-inline-adder-field adaptive>
+            <oui-field label="Field 2" size="m">
+                <label class="oui-select">
+                    <select name="field2" ng-model="field2" class="oui-select__input" required>
+                        <option value="" disabled selected>Select the OS</option>
+                        <option value="freebsd">FreeBSD</option>
+                        <option value="linux">Linux</option>
```

Comment:
As you may know, **Linux** is not an operating system. You probably mean a **GNU/Linux distributions** right? :)

Reply:
I'll update the example. It comes from the past contribution.